southpark-search/README.md

```diff
@@ -230,21 +230,21 @@ python app.py -t index -n 10000
   <img src=".github/index-demo.png?raw=true" alt="index flow console output">
 </p>
 
-</details> 
+</details>
 
-With the Flows, we now can write the codes to run the Flow. For indexing, we start with defining the Flow with YAML file. Afterwards, the `build()` function will do the magic to construct Pods and connect them together. Then the `IndexRequest` will be sent to the flow by calling the `index()` function. 
+With the Flows, we can now write the code to run the Flow. For indexing, we start by defining the Flow with a YAML file. Afterwards, the `build()` function will do the magic to construct Pods and connect them together. After that, the `IndexRequest` will be sent to the flow by calling the `index()` function.
 
-```python        
+```python
 def main(num_docs):
     flow = Flow().load_config('flow-index.yml')
     with flow.build() as fl:
-    	 data_fn = os.path.join('/tmp/jina/southpark', 'character-lines.csv')
+       data_fn = os.path.join('/tmp/jina/southpark', 'character-lines.csv')
```

Comment:
Here is an indentation error.

Reply:
Vim somehow messed up the indent when I was editing! Fixed now